letsencrypt/rootfs/etc/services.d/lets-encrypt/run

```diff
@@ -96,6 +96,13 @@ elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-njalla" ]; then
     bashio::config.require 'dns.njalla_token'
     PROVIDER_ARGUMENTS+=("--authenticator" "certbot-dns-njalla:dns-njalla" "--certbot-dns-njalla:dns-njalla-credentials" /data/dnsapikey "--certbot-dns-njalla:dns-njalla-propagation-seconds" "${PROPAGATION_SECONDS}")
 
+# IONOS
+elif [ "${CHALLENGE}" == "dns" ] && [ "${DNS_PROVIDER}" == "dns-ionos" ]; then
+    bashio::config.require 'dns.ionos_endpoint'
+    bashio::config.require 'dns.ionos_prefix'
+    bashio::config.require 'dns.ionos_secret'
+    PROVIDER_ARGUMENTS+=("--authenticator" "${DNS_PROVIDER}" "--${DNS_PROVIDER}-credentials" /data/dnsapikey)
+
```

Comment:
```suggestion
```
Don't do schema validation here. If you want to do validation like this you have to do it in the `cont-init.d` file. The problem with doing validation in the run script is that if this script fails then s6 will try to restart it causing the same validation errors over and over again.